scripts/installer/install.sh

```diff
@@ -283,16 +281,27 @@ platform_install_bootloader_entry()
     /bin/true
 }
 
+onie_platform=$(onie-sysinfo -p)
+
+# <arch>-<vendor>-<platform>-<revision>
+onl_platform="$(onie-sysinfo -p | tr '_' '-')"
+
+# <arch>-<vendor>-<platform>
+onl_baseplatform="${onl_platform%-r*}"
+
+[ -n $DEBUG ] && echo "DEBUG: onl_platform=${onl_platform}"
+[ -n $DEBUG ] && echo "DEBUG: onl_baseplatform=${onl_baseplatform}"
+
 cd $(dirname $0)
-. ./machine.conf
+
 # platform.sh may override dummy functions above (e.g., platform_check)
 . ./platform.sh
 
-echo "BISDN Linux Installer: platform: $platform"
+echo "BISDN Linux Installer:"
```

Comment:
The colon doesn't make any sense here anymore

```suggestion
echo "BISDN Linux Installer"
```

Reply:
Done.